app/src/main/java/com/infomaniak/mail/data/models/thread/Thread.kt

```diff
@@ -61,7 +61,7 @@ class Thread : RealmObject {
     var to: RealmList<Recipient> = realmListOf()
     var subject: String? = null
     @SerialName("date")
-    private var _date: RealmInstant = RealmInstant.MAX
+    var _date: RealmInstant = RealmInstant.MAX
```

Comment:
When the name of a variable starts with `_` it is only for private variables

Reply:
Done.